helm/wts/templates/wts-oidc.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.oidc_job_enabled }}
+{{- if and (.Values.oidc_job_enabled) (.Values.externalSecrets.createWtsOidcClientSecret) }}
```

Comment:
I might have end-of-day brain, but I think this will create the job only if we ARE creating the external secret? I think that's the opposite of the logic we want.

Reply:
It's a little confusing, but we typically set the createBlahBlahSecret variable to true if we want to create the secret with helm, not if we want it to be managed through external secrets. So I think this should only do it if we want to enable the job and if we want to create the secret with helm(not through external secrets).